File: ruby/lib/svix/api_internal/destination.rb
# frozen_string_literal: true
# This file is @generated

require "net/http"

module Svix
  module ApiInternal
    class Destination
      attr_accessor :autoconfig
      def initialize(client)
        @client = client
        @autoconfig = DestinationAutoconfig.new(client)
      end
    end
  end
end

File: python/svix/api_internal/destination.py
# This file is @generated
from .common import ApiBaseAsync, ApiBaseSync
from .destination_autoconfig import (
    DestinationAutoconfig,
    DestinationAutoconfigAsync,
)


class DestinationAsync(ApiBaseAsync):
    @property
    def autoconfig(self) -> DestinationAutoconfigAsync:
        return DestinationAutoconfigAsync(self._client, self._httpx_client)


class Destination(ApiBaseSync):
    @property
    def autoconfig(self) -> DestinationAutoconfig:
        return DestinationAutoconfig(self._client, self._httpx_client)
